### DISCUSSION TITLE
Multiple Pi's on the same "network"?

### DISCUSSION THREAD

**Discussion by cspradlin** [zwave-js/zwave-js#2499] [Q&A] [answered]
Hello!  I couldn't find a similar question with the search tool, sorry if it's a repeat.

I've got a fairly expansive area to cover with a detached workshop/outbuilding--I have a wireless bridge to expand my LAN/WLAN, but it's well out of range of Z-WAVE due to construction materials and distance.  I'm looking  to expand my coverage and install some sensors, using a second Pi running ZwaveJS/ZwaveJS2mqtt just like the first in the main house.

My main question is, is it possible to utilize the websocket server when using two Pis?  There is no way to configure that in HomeAssistant, which makes sense since the first Pi is acting as the actual ZwaveJS server, and as far as I can tell there is no way to configure this in the GUI available with the Docker image.  Is there a way to do this manually in the configuration files or with commands, or do I need to swap over to the MQTT implementation if I intend to use multiple "hub"-type devices?

**Comment by kpine** [comment, marked as answer]
> My main question is, is it possible to utilize the websocket server when using two Pis? There is no way to configure that in HomeAssistant,

All you have to do is add a second instance of the zwave_js integration, giving it the ws URL of the second host. HA has supported multiple zwave-js servers since the first release.

![image](https://user-images.githubusercontent.com/791794/116789779-7073fb00-aa65-11eb-897c-ae8df3811331.png)

**Reply by cspradlin** [reply]
Turns out I'm a fool who way overthought the process!  I was looking for configurations, including just telling one pi to report to the other, but yeah just manually adding another integration instance worked right off the bat.

Now to get all the automations worked out and set up!
